src/pilot/register_overcloud.py

```diff
@@ -257,18 +262,14 @@ def _register_node(self, role, node_ip):
                 sys.exit(1)
 
             # The node isn't registered, so register it
-            self.logger.info("Registering {} {} with CDN".format(role,
-                                                                 node_ip))
+            self.logger.info("Registering {} {}".format(role,
+                                                        node_ip))
 
-            # If we're using an activation key, then construct the args for
-            # that
+            # If we're using satellite, then construct the args for that
             cred_args = ""
-            if self._using_activation_key(role):
```

Comment:
This change removes support for using activation keys with CDN.  Do we really want to do that?